conf.d/main

```diff
@@ -29,21 +32,22 @@ a2ensite elgg
 a2enmod rewrite
 
 # start services
-/etc/init.d/mysql start
-/etc/init.d/apache2 start
+service mysql start
+service apache2 start
 
 # setup the database
 MYSQL_BATCH="mysql --user=root --password=$MYSQL_PASS --batch"
 MYSQL_ADMIN="mysqladmin --user=root --password=$MYSQL_PASS"
 
 $MYSQL_ADMIN create $DB_NAME
 $MYSQL_BATCH --execute "grant all privileges on $DB_NAME.* to $DB_USER@localhost identified by '$DB_PASS'; flush privileges;"
+$MYSQL_BATCH --execute "INSTALL SONAME 'ha_archive';"
 
 # cli based installation
-chown www-data:www-data $WEBROOT
-chown www-data:www-data $WEBROOT/engine
-INSTALLER=$WEBROOT/install/cli/sample_installer.php
-
+rm $WEBROOT/.htaccess
```

Comment:
What was the rationale for removing the `.htaccess` file? 

You probably already know, but `.htaccess` files are Apache config snippets that only apply to the directory hierarchy they appear in. E.g. `/var/www/some-app/.htaccess` will apply to the whole `/var/www/some-app/` directory (and any sub-directories). FWIW, the effect would be exactly the same if you put the contents of the `.htaccess` file in the Apache site config file within a `<Directory /var/www/some-app>` block. Apache docs suggest that there is a performance hit when using `.htaccess` files, although I would imagine it would be fairly minimal.

Another note worth making is that `.htaccess` files will be ignored unless the Apache site config it has a `Directory` directive block which includes `AllowOverride All` (default is `AllowOverride None`). As the name suggests, that allows the `.htaccess` config to override the Apache site config.

I note that the existing Apache site conf (i.e. `elgg.conf`) already contains `AllowOverride All`, so simply leaving the `.htaccess` file there will allow that to be processed. However, perhaps that was why you removed it? Perhaps it was causing issues? If so, it's probably worth working out why and fixing it. You could even move the relevant contents into the Apache config file if you wanted...

Reply:
Appreciate the explanation but I do know what .htaccess files are 😄. This was a while back, so I don't remember explicitly why I've done this but I'm happy to go back and have a look :)

Reply:
So the `.htaccess` file was removed because the elgg installer will attempt to create one itself which will then fail if the file already exists.

However you're comment has alluded me to another issue that is present in this pull request. (Timezone is not being set) so don't close yet! :)

Reply:
Actually that was a false issue, although the installer complains about the timezone being incorrect, there is no section in the installer to set it, and only once the settings file has been created (by the installer) can the timezone be set.

Reply:
Hey @OnGle - apologies on my `.htaccess` ramble. I should have know that you would know what they are (and are capable with google if not). Hopefully I didn't offend. 

FWIW It was Friday afternoon and I think the `-vvv` switch must have been called somewhere along the line... :smile: 

Anyway, great work on your quick followup. I'll merge now. Cheers.